tests/benchdnn/dnnl_common.hpp

```diff
@@ -309,7 +310,7 @@ int check_dnnl_status(dnnl_status_t status, const prb_t *prb, res_t *res) {
         case dnnl_unimplemented: {
             // Unconditionally set all Nvidia backend unimplemented cases as
             // not supported.
-            if (is_nvidia_gpu() || is_amd_gpu()) {
+            if (is_nvidia_gpu() || is_amd_gpu() || is_intel_gpu()) {
```

Comment:
It should not be done the same way for Nvidia backend just because Nvidia and AMD has very limited feature support and it would be unprecedented effort to figure-cutting all the cases that are supported or not for those backends.
Intel backend support most or all features, that's why there's a dedicated section to figure-cutting known unsupported cases which would become supported eventually.

Intel backend has two tasks to solve: (a) being correct on supported problems; (b) catch unimplemented cases and make sure those are intentional unsupported cases.
For Nvidia/AMD backends there's only task (a) to be solved, thus, you see the solution to skip whatever is unimplemented.

Hope it clears the water.

I propose do nothing about those blocked layouts as they soon die out themselves.

Reply:
We could leave them as they are now, basically what you are proposing. The thing is that on benchdnn we will get an UNIMPLEMENTED and not a clear skip case no supported message. If you think it is OK it will be also good for me. Thanks.